src/pages/organize/[orgId]/projects/[campId]/events/[eventId]/participants.tsx

```diff
@@ -53,6 +54,13 @@ const ParticipantsPage: PageWithLayout<ParticipantsProps> = ({
               <Grid item md={8} xs={12}>
                 <ParticipantSummaryCard model={dataModel} />
               </Grid>
+              <Grid item md={4} xs={8}>
```

Comment:
I think `xs` should be `12`, so that it occupies the full width on small screens.